EIPS/eip-2390.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 2390
+title: Geo-ENS
+author: James Choncholas <jchoncholas@gmail.com>
```

Comment:
```suggestion
author: James Choncholas (@james-choncholas)
```
Ideally, there is at least one author with a GitHub handle on the EIP so the auto-merge bot can work.